### PR TITLE
Lydia see correct answer

### PR DESCRIPTION
**Pull request by Lydia-GG**
# Pull Request

> :arrow_right: Describe your pull request. Make sure to detail all changes and what the purpose of the PR is. PRs should be concise and only contain files related to the PR!

## Checklist

Complete these things in order to submit your PR for review:

- [ ] :mag: I have manually checked the code in my changes
- [ ] :judge: I have tested my changes
- [ ] :pencil: I have provided comments in code to explain my changes

### CODE REVIEW

File: src/views/questionViews.js [hunk 1]
```diff
@@ -28,6 +28,15 @@ export const createQuestionElement = (question) => {
...
     const answer = createAnswerElement(question.answers[answerKey]);
     answerContainer.appendChild(answer);
+    
+    // add green background with correct answer and red background with wrong answer
+    answer.addEventListener("click", ()=>{
+      if(answerKey === question.correct){
+        answer.style.cssText = "background-color: green; color: white;"
+      }else{
+        answer.style.cssText = "background-color: red; color: white;"
+      }
```
Comment (cgduncan7) [inline src/views/questionViews.js:38]: I don't think this is the correct place for this code! In the `README` for views it states that they will never use events! This is using an event 😄 

Where should you *handle* this event?

File: src/views/questionViews.js [hunk 2]
```diff
@@ -10,7 +10,7 @@ import { createDOMElement } from '../utils/DOMUtils.js';
 export const createAnswerElement = (answerText) => {
   const answerElement = createDOMElement('li');
   answerElement.innerText = answerText;
-
+  answerElement.style.cursor = "pointer";
```
Comment (cgduncan7) [inline src/views/questionViews.js:13]: ✨

**Review by cgduncan7 — CHANGES_REQUESTED** [review]

File: src/handlers/questionHandlers.js [hunk 3]
```diff
@@ -20,3 +20,13 @@ export const handleNextQuestion = () => {
 
   showCurrentQuestion();
 };
+
+export const handleCheckAnswer = (selectedAnswer) => {
```
Comment (cgduncan7) [inline src/handlers/questionHandlers.js:24]: What is `selectedAnswer`?

File: src/listeners/questionListeners.js [hunk 4]
```diff
@@ -1,7 +1,12 @@
...
 
 import { handleNextQuestion } from '../handlers/questionHandlers.js';
+import { handleCheckAnswer } from '../handlers/questionHandlers.js';
 
 export const nextQuestion = () => {
   handleNextQuestion();
 };
+
+export const checkAnswer = () => {
+  handleCheckAnswer();
```
Comment (cgduncan7) [inline src/listeners/questionListeners.js:11]: `handleCheckAnswer` takes one argument of `selectedAnswer` but you provide no argument!

File: src/views/questionViews.js [hunk 5]
```diff
@@ -28,6 +30,15 @@ export const createQuestionElement = (question) => {
...
     answerContainer.appendChild(answer);
+    
+    // add green background with correct answer and red background with wrong answer
+    // answer.addEventListener("click", ()=>{
+    //   if(answerKey === question.correct){
+    //     answer.style.cssText = "background-color: green; color: white;"
+    //   }else{
+    //     answer.style.cssText = "background-color: red; color: white;"
+    //   }
+    // })
```
Comment (cgduncan7) [inline src/views/questionViews.js:41]:
```suggestion
```

File: src/handlers/questionHandlers.js [hunk 6]
```diff
@@ -20,3 +20,16 @@ export const handleNextQuestion = () => {
...
+const currentQuestion = quizData.questions[quizData.currentQuestionIndex];
+let clickedAnswer = selectedAnswer.innerText
+
+let currentCorrectAnswer = currentQuestion.answers[currentQuestion.correct];
```
Comment (cgduncan7) [inline src/handlers/questionHandlers.js:28]: Nice one! This is perfect

File: src/listeners/questionListeners.js [hunk 7]
```diff
@@ -1,7 +1,11 @@
...
 
-import { handleNextQuestion } from '../handlers/questionHandlers.js';
+import { handleNextQuestion ,handleCheckAnswer } from '../handlers/questionHandlers.js';
 
 export const nextQuestion = () => {
   handleNextQuestion();
 };
+
+export const checkAnswer = (event) => {
+  handleCheckAnswer(event.target);
```
Comment (cgduncan7) [inline src/listeners/questionListeners.js:10]: 👍

**Review by cgduncan7 — DISMISSED** [review]

**Comment by cgduncan7** [comment]
@Lydia-GG nice work 👏 now you need to resolve these merge conflicts! In VS Code you should:
```
git checkout master // checkout master branch
git pull // update master branch
git checkout Lydia-see-correct-answer // checkout your WIP branch
git merge master // merge changes from master into your branch
```

Then you will see conflicts in the above files ^

If you need help, let me know!

**Comment by Lydia-GG** [comment]
> @Lydia-GG nice work 👏 now you need to resolve these merge conflicts! In VS Code you should:
> 
> ```
> git checkout master // checkout master branch
> git pull // update master branch
> git checkout Lydia-see-correct-answer // checkout your WIP branch
> git merge master // merge changes from master into your branch
> ```
> 
> Then you will see conflicts in the below files vvvv
> 
> If you need help, let me know!

**Comment by Lydia-GG** [comment]
> > @Lydia-GG nice work 👏 now you need to resolve these merge conflicts! In VS Code you should:
> > ```
> > git checkout master // checkout master branch
> > git pull // update master branch
> > git checkout Lydia-see-correct-answer // checkout your WIP branch
> > git merge master // merge changes from master into your branch
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Then you will see conflicts in the below files vvvv
> > If you need help, let me know!

**Review by cgduncan7 — APPROVED** [review]
👏 🎆

**Pull request merged**